bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/3D/Transformation/Rotation/Quaternion.cpp

```diff
@@ -70,8 +70,8 @@ inline void OpenSpaceToolkitMathematicsPy_Geometry_3D_Transformation_Rotation_Qu
         )
 
         .def("is_defined", &Quaternion::isDefined)
-        .def("is_unitary", &Quaternion::isUnitary)
-        .def("is_near", &Quaternion::isNear, arg("quaternion"), arg("angular_tolerance"))
+        .def("is_unitary", &Quaternion::isUnitary, arg("norm_tolerance"))
+        .def("is_near", &Quaternion::isNear, arg("quaternion"), arg("angular_tolerance") = Real::Epsilon()))
```

Comment:
I forget, do we often add default args in the bindings when they're not in the cpp? If not then maybe don't add it here because it can promote a difference in usage between cpp and python for the same method

Reply:
Sorry, I pushed too fast and added to the function `is_near` that's my bad, I reverted